### DISCUSSION TITLE
致新来的你

### DISCUSSION THREAD

**Discussion by adoyle-h** [just-talks/tech-talks#1] [0.公告]
欢迎！

在这个论坛我们只围绕技术话题进行分享、讨论、提问、解答。
这里没有人生感言，没有政治立场，没有职场八卦，没有招聘信息，没有二手交易。
希望在这里，我们能够分享有趣的事物，交流想法，获得灵感，结识朋友，提升自己。

## 导航

- [特点](https://github.com/just-talks/tech-talks#特点)
- [版块目录](https://github.com/just-talks/tech-talks#版块目录)
- [使用方法](https://github.com/just-talks/tech-talks#使用方法)
    - [参与讨论](https://github.com/just-talks/tech-talks#参与讨论)
    - [创建讨论](https://github.com/just-talks/tech-talks#创建讨论)
    - [搜索讨论](https://github.com/just-talks/tech-talks#搜索讨论)
    - [订阅讨论](https://github.com/just-talks/tech-talks#订阅讨论)
    - [提问](https://github.com/just-talks/tech-talks#提问)
    - [回答与评论](https://github.com/just-talks/tech-talks#回答与评论)
    - [投诉](https://github.com/just-talks/tech-talks#投诉)
    - [其他](https://github.com/just-talks/tech-talks#其他)
- [用户守则](https://github.com/just-talks/tech-talks#用户守则)
- [管理员列表](https://github.com/just-talks/tech-talks#管理员列表)
- [版权声明](https://github.com/just-talks/tech-talks#版权声明)

其他请看 [README](https://github.com/just-talks/tech-talks/blob/master/README.md)。

## 帮助建设论坛

- 给本项目加一颗 ⭐️
- 将本论坛推荐给朋友。
- 参与讨论。
- 发帖分享思考或者优秀的作品。

**Comment by jiacai2050** [comment]
给楼主的想法点赞。

有个小建议，现在分类太多了，我的建议是都去掉，就保留这一个主仓。等后面人多了，在考虑分类问题。

**Reply by kalxd** [reply]
夹菜兄之言有理啊。

**Reply by adoyle-h** [reply]
emmm，那就在主仓库发帖吧，我之后再整理
